.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: "Mark/Close Stale Issues/PRs"
+on:
+  schedule:
+  - cron: "* */1 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v1.1.0
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: |-
+          This issue is stale because it has been open 180 days with no activity. Remove stale label or comment on the issue or it will be closed in 30 days.
+        stale-pr-message: |-
+          This pull request is stale because it has been open 180 days with no activity. Remove stale label or comment on the pull request or it will be closed in 30 days.
+        days-before-stale: 180
+        days-before-close: 30
```

Comment:
Do we want to change these values?

Reply:
They seem reasonable to me.  But I would prefer only a 14 day grace period myself.

Reply:
Fixed in [force-push](https://github.com/eclipse/omr/compare/02fed63e6206221d73bd82917a10bc0ac274c3f6..94f6fb6ea30c69ca960f027c5b75e14f2cf63186). I've changed it to 14 days.